tests/wpt/mozilla/meta/mozilla/htmlanchorelment-search-attribute.html.ini

```diff
@@ -0,0 +1,3 @@
+[htmlanchorelment-search-attribute.html]
+  type: testharness
+  expected: OK
```

Comment:
Yes, remove this. By default, tests are expected to PASS (or OK), so if there's no .ini file, the test framework will expect the test file to pass.